CSVInput/medium_file_parent_sales.csv

```diff
@@ -0,0 +1,32 @@
+Sale_ID, Date, Total_Price, Quantity_of_Items, Buyer, Seller
+1,2022/10/12,19563.9,3,Stephen,Vinny
```

Comment:
I think we have an issue here. Our Date class stores dates as M/D/Y and the sales save() function just prints that out. I see that the dates are read in the way you have them, but we will rewrite this with the improper format the first time we read it in and quit our program and then it will be backwards. Right?